test/functional/vimscript/special_vars_spec.lua

```diff
@@ -130,6 +130,12 @@ describe('Special values', function()
     eq("v:false", eval('"" . v:false'))
   end)
 
+  it('work with ?? (falsy operator)', function()
+    eq(true, eval('v:true ?? 42'))
+    eq(42, eval('v:false ?? 42'))
+    eq(42, eval('v:null ?? 42'))
+  end)
+
```

Comment:
Other tests for falsy operator can be found here. Wouldn't it be better to put them all in the same place?
https://github.com/neovim/neovim/blob/3ac952d4e27f4e2454332a730310316fe13fd4a3/test/old/testdir/test_expr.vim#L64-L87

Reply:
Not really.